tests/wpt/mozilla/tests/mozilla/bluetooth/getCharacteristic/disconnect-called-during.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+var server;
+promise_test(t => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.heart_rate);
+    return window.navigator.bluetooth.requestDevice({
+        filters: [{services: [heart_rate.name]}],
+        optionalServices: [generic_access.name]
+    })
+    .then(device => device.gatt.connect())
+    .then(gattServer => {
```

Comment:
You can use the `gattServer` without a plus `var` like (this)[https://github.com/szeged/servo/blob/test-api-impl/tests/wpt/mozilla/tests/mozilla/bluetooth/disconnect/connect-disconnect-twice.html#L12]